src/Components/LifecycleChart/LifecycleChart.tsx

```diff
@@ -296,6 +298,31 @@ const LifecycleChart: React.FC<LifecycleChartProps> = ({
     };
   }, [updatedLifecycleData.length]);
 
+  // Create custom tooltip function that safely handles the data
+  const getTooltipLabel = (point: any, index: number, points: any[]): string => {
```

Comment:
```suggestion
  const getTooltipLabel = (point: any): string => {
```
Can be removed, they aren't used.

Also I think it would be great to have type definiton for point (as I remember we should avoid the `any` as much as we can), but from the debugger I see it's some internal vornoi object and defining it might be complicated, so I'm fine with `any` (as the type is mostly checked inside of the function).
![Screenshot From 2025-04-04 09-13-45](https://github.com/user-attachments/assets/7401f66a-899a-4404-95fc-20909f122698)